Solutions/Corvus.SpecFlow.Extensions/Corvus/SpecFlow/Extensions/Internal/ProcessOutputHandler.cs

```diff
@@ -90,6 +95,28 @@ public string StandardErrorText
             }
         }
 
+        /// <inheritdoc />
+        public void ClearAllOutput()
+        {
+            lock (this.standardOutput)
```

Comment:
Similarly it feels like we should have timeouts on these. I also worry about locking these actual objects rather than proxies for them. Deadlocks can occur if the implementation decides to take locks on itself internally at some point.

Reply:
Raised https://github.com/corvus-dotnet/Corvus.SpecFlow.Extensions/issues/79